lib/prefix.h

```diff
@@ -64,12 +81,14 @@ struct ethaddr {
 #define PREFIX_LEN_ROUTE_TYPE_5_IPV6 (30*8)
 
 typedef struct esi_t_ {
```

Comment:
I understand we shouldn't be adding new typedefs. This should be added to the list of typedfs to purge when the dust settles.